docs/products/kafka/kafka-connect/howto/jdbc-sink.rst

```diff
@@ -0,0 +1,273 @@
+
+If you have a topic named ``students`` containing data in Avro format with the schema stored in the schema registry provided by `Karapace <https://help.aiven.io/en/articles/5651983>`_ with the following structure:
+
+.. code-block:: text
+
+    key: {"student_id": 1234}
+    value: {"student_name": "Mary", "exam": "Math", "exam_result":"A"} 
+
+You can sink the ``students`` topic to MySQL with the following connector configuration, after replacing the placeholders for ``DB_HOST``, ``DB_PORT``, ``DB_NAME``, ``DB_SSL_MODE``, ``DB_USERNAME``, ``DB_PASSWORD``, ``APACHE_KAFKA_HOST``, ``SCHEMA_REGISTRY_PORT``, ``SCHEMA_REGISTRY_USER`` and ``SCHEMA_REGISTRY_PASSWORD``:
+
+.. code-block:: json
+
+    {
+        "name": "sink_students_avro_schema",
+        "connector.class": "io.aiven.connect.jdbc.JdbcSinkConnector",
+        "topics": "my_pgnordics2022_pgsource.public.pasta",
+        "connection.url": "jdbc:mysql://DB_HOST:DB_PORT/DB_NAME?ssl-mode=DB_SSL_MODE",
+        "connection.user": "DB_USERNAME",
+        "connection.password": "DB_PASSWORD",
+        "insert.mode": "upsert",
+        "table.name.format": "students",
+        "pk.mode": "record_key",
+        "pk.fields": "student_id",
+        "auto.create": "true",
+        "auto.evolve": "true",
+        "delete.enabled": "true",
+        "key.converter": "io.confluent.connect.avro.AvroConverter",
+        "key.converter.schema.registry.url": "https://APACHE_KAFKA_HOST:SCHEMA_REGISTRY_PORT",
+        "key.converter.basic.auth.credentials.source": "USER_INFO",
+        "key.converter.schema.registry.basic.auth.user.info": "SCHEMA_REGISTRY_USER:SCHEMA_REGISTRY_PASSWORD",
+        "value.converter": "io.confluent.connect.avro.AvroConverter",
+        "value.converter.schema.registry.url": "https://APACHE_KAFKA_HOST:SCHEMA_REGISTRY_PORT",
+        "value.converter.basic.auth.credentials.source": "USER_INFO",
+        "value.converter.schema.registry.basic.auth.user.info": "SCHEMA_REGISTRY_USER:SCHEMA_REGISTRY_PASSWORD"
+    }
+
+The configuration file contains the following peculiarities:
+
+* ``"topics": "students"``: setting the topic to sink
+* ``"pk.mode": "record_key"``: the connector is using the message key to set the target database key
+* ``"pk.fields": "student_id"``: the connector is using the field ``student_id`` on the message key to set the target database key
+* ``"delete.enabled": "true"``: the connector is enabling deletes on tombstones
+* ``key.converter`` and ``value.converter``: defining the Avro data format with ``io.confluent.connect.avro.AvroConverter``, the URL, and credentials to connect to the `Karapace <https://help.aiven.io/en/articles/5651983>`_ schema registry
+
+The connector will automatically create ``"auto.create": "true"`` a table in the target MySQL database called ``students`` with ``student_id``, ``student_name``, ``exam`` and ``exam_result`` as columns and populate it with the data coming from the ``students`` Apache Kafka topic.
```

Comment:
I think it would be nice to have a final conclusion like "This is how you migrated Kafka connect.."

Reply:
We don't have it for other connectors, and the final sentence already shows what it's doing... I'm not really sure about what an extra sentence would give? but open for discussion

Reply:
I re-read and I understand now that this is an example more like a general topic.